contract_fixed_discount/views/abstract_contract_line.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--FORM view-->
+    <record
+        id="contract_abstract_contract_line_add_discount_fixed_form_view"
+        model="ir.ui.view"
+    >
+        <field
+            name="name"
+        >contract.abstract.contract.line form view (in contract)</field>
+        <field name="model">contract.abstract.contract.line</field>
+        <field
+            name="inherit_id"
+            ref="contract.contract_abstract_contract_line_form_view"
+        />
+
+        <field name="arch" type="xml">
+            <field name="discount" position="after">
+                <field name="discount_fixed" optional="hide" />
```

Comment:
testing on runboat :
* this looks like to manage the popup form view which which don't care about optional
* so we should add it on the tree view in the contract form view
* we should probably use same group as the discount fields
* I think we should change the group to rely on the same one present on sale order: `groups="product.group_discount_per_so_line"`

Reply:
Test on runboat: thanks :heart: